fix: unstable tests

shhdgit opened the pull request:
- fix unstable tests
- close #1174

breezewish commented on ui/cypress/integration/topsql/topsql.spec.ts:
```diff
@@ -167,13 +167,11 @@ skipOn(Cypress.env('TIDB_VERSION') !== 'nightly', () => {
         cy.wait('@getTopsqlSummary')
```
Maybe you should test with seconds with sub-seconds for better coverage, e.g. now * 1000 + 456.

BTW there is indeed time range problems in Top SQL related with subseconds, as the displayed time range will vary within 59s, 60s, 61s when using the brush tool.

breezewish commented on .github/workflows/test.yaml:
```diff
@@ -77,10 +77,10 @@ jobs:
         include:
           - feature_version: 6.0.0
-            tidb_version: nightly
+            tidb_version: latest
```
latest and nightly are different. latest should be pointing to stable release only I guess.

breezewish replied:
How about introducing both latest and nightly to our matrix?

IMO currently the source of unstable comes from our tests, instead of nightly.

shhdgit replied:
Make sense.

I found some unstable cases when restarting tiup playground nightly in m1. It can be resolved by the cypress default env variable.

breezewish reviewed (commented):
It's interesting that the image size is also changed.. I'm not sure where it comes from.

shhdgit commented:
> It's interesting that the image size is also changed.. I'm not sure where it comes from.

Oh, I updated the snapshots for the stability of screenshots. It's will be clipped, if it is not the default viewport configuration.

breezewish commented:
Merging, cause v6.0.0 branch looks to be stabled.
